doc/features.rst

```diff
@@ -7,8 +7,8 @@ Features
 * Includes a rich `library of composable elements <http://www.holoviews.org/Tutorials/Elements.html>`_ that can be overlaid, nested and positioned with ease.
 * Supports `rapid data exploration <http://www.holoviews.org/Tutorials/Exploring_Data.html>`_ that naturally develops into a `fully reproducible workflow <http://www.holoviews.org/Tutorials/Exporting.html>`_.
 * Create interactive visualizations that can be controlled via widgets or via custom events in Python using the 'streams' system. When using the bokeh backend, you can use streams to directly interact with your plots.
-* Rich semantics for `indexing and slicing of data in arbitrarily high-dimensional spaces <http://www.holoviews.org/Tutorials/Sampling_Data.html>`_.
-* Plotting output using the `Matplotlib <http://www.holoviews.org/Tutorials/Elements.html>`_, `Bokeh <http://www.holoviews.org/Tutorials/Bokeh_Elements.html>`_, and `plotly <http://plot.ly/>`_ backends.
+* Rich semantics for `indexing and slicing of data in arbitrarily high-dimensional spaces <user_guide/Indexing_and_Selecting_Data.html>`_.
```

Comment:
Let us use relative paths for everything if it works. And for external sites let's use https.

Reply:
The new relative paths are not exact 1:1 mappings as some of the content in the old `Tutorials` directory have been modified and names changed on moving to `User Guide` dir.

Reply:
I saw that and don't have any problems with it, as it should ideally have been done when the things were removed/moved.